source/mvgApi.js

```diff
@@ -2,26 +2,28 @@
 const cheerio = require("cheerio");
 const request = require("request");
 const Line = require("./Line");
+const Station = require("./Station");
 const iconv = require("iconv-lite");
 const defaultApi = "https://www.mvg-live.de/ims/dfiStaticAuswahl.svc";
 
 /**
- * Get the departures for a certain station.
  *
- * @station String The name of the station you want to get the departures for.
- * @options Array An array of the types of transportation you want to have shown.
- *          Can be one or more of the following:
- *          "ubahn", "sbahn", "tram", "bus"
- *          If not provided, departures for all kinds of transit are shown.
- * @apiUrl  String for custom API endpoint. If not provided, defaultApi is used
+ * @param {string} station - The name of the station you want to get the departures for.
+ * @param {Array} options - An array of the types of transportation you want to have shown.
+ *                 Can be one or more of the following:
+ *                 "ubahn", "sbahn", "tram", "bus"
+ *                 If not provided, departures for all kinds of transit are shown.
+ * @param {string} apiUrl - String for custom API endpoint. If not provided, defaultApi is used
+ * @returns {Promise}
+ * @TODO: refactor function to accept Station Instance as station
```

Comment:
Not sure what the benefit of refactoring it this way would be...;)

Reply:
If suggestions return Station-Objects, we could use them (also using identifiers instead of name strings)

Depends on future implementation of the "new mvg-API"